src/util/generatePreview.js

```diff
@@ -122,6 +124,12 @@ class PreviewGenerator {
     this._previewHead.appendChild(styleTag);
   }
 
+  _addRefreshTimestamp(timestamp) {
+    const comment = this.previewDocument.createComment(`Last refresh on: ${String(new Date(timestamp))}`);
+    console.log(comment);
+    this.previewBody.prepend(comment);
```

Comment:
All else being equal I’d probably put this at the end although obviously doesn’t matter too much either way…